app/javascript/components/DelegationContactCard.jsx

```diff
@@ -8,26 +8,28 @@ function DelegationContactCard({
   email,
   handleClickChangeContact,
   handleClickRemoveContact,
+  showHeader = true,
+  showChangeBtn = true,
   showRemoveContactBtn = true
 }) {
   return (
     <div>
-      <p className="delegation__label">Handing off question to</p>
+      {showHeader && <p className="delegation__label">Handing off question to</p>}
```

Comment:
i cant use an if statement without making a function so i can change it to () => { if (showHeader) {...}} but this is kind of wordy imo

Reply:
you can do `{showHeader ? (<p className...>) : null}` (or you can render something else instead of null if need be)
This basically translates to
```
if (showHeader) {
 render <p className....>
}
else{
 render nothing
}
```

Feel free to DM me if this doesn't make sense or if you run into problems!